docs/nf_templates/4_anatomy.md

```diff
@@ -0,0 +1,37 @@
+# Anatomy of the nf-core template
+
+The nf-core pipeline template comes packed with a lot of files and folders which can be overwhelming.
+
+Fortunately, a complete understanding isn't required to get started.
+
+<figure class="excalidraw">
+--8<-- "docs/nf_templates/img/4_template.excalidraw.svg"
```

Comment:
I find this pretty difficult to look at. Suggest to either remove, or have as a "photo of a screen" type thing, where it's (a) smaller and (b) clear that the reader isn't meant to interpret anything it says.

Reply:
Even an image of the terminal or something could work, eg:

![codetoimg-snippet (1)](https://github.com/nextflow-io/training/assets/465550/5162ab0a-7e2a-40a1-93af-eadb6ae8578b)
![codetoimg-snippet](https://github.com/nextflow-io/training/assets/465550/6f581f4c-cb4b-4dcf-b933-93d433e0731c)

Reply:
Yeah, I see your point. I'll swap it out with a screenshot.